src/main/java/leetCode/TicketPath.java

```diff
@@ -4,7 +4,37 @@
 
 public class TicketPath {
     private static HashMap<String, LinkedHashSet<String>> flights;
+    private static HashMap<String, String> flights2;
 
+//    No use of helper functions
+    public static String ticketPathDecoderV2(Stack<Ticket> stackOfTickets){
+        flights2 = new HashMap<>();
+        ArrayList<String> arrivals  = new ArrayList<>();

```

Comment:
nit: please change to `List<String> arrivals = new ArrayList<>();`. You want to code to [interface](https://medium.com/@sanchezdale/oop-good-practices-coding-to-the-interface-baea84fd60d3)

Reply:
? not sure what ya mean :)
should I change it to `ArrayList<Strings> arrivals = new ArrayList<>();`
and reading the article linked

Reply:
should be as described in my comment 😄 . It's generally a good idea to provide higher level abstractions instead of providing concrete implementations. Higher level abstraction: `List<T>`, concrete implementation `ArrayList<>`, please see more examples [here](https://www.geeksforgeeks.org/list-interface-java-examples/)

Reply:
DUH (slaps face) didn't read my code first, lol. I put it right back to what I had. I thought being more specific was a better practice.